src/style/normalize.css

```diff
@@ -0,0 +1,14 @@
+body,
+h2,
+div,
+span,
+a {
+  margin: 0;
+}
+
+.product-card {
+  font-family: "Roboto", Arial, Helvetica, sans-serif;
+  font-size: 10px;
+
+  box-sizing: border-box;
```

Comment:
move that to your main `product-card` class

Reply:
That's not a proper way to implement it
As for me, `normalize.css` shouldn't contain styles for what is meant to be a BEM-block